lbryum/commands.py

```
@@ -165,9 +171,9 @@ def update_password(self, new_password, update_keyring=True):
         with self.wallet.storage.lock:
             self.new_password = new_password
             self.wallet.update_password(self._password, self.new_password)
-            if update_keyring:
+            if update_keyring and self._keyring is not None:
```

if update_keyring is True and there isn't a keyring available, it should probably return an error rather than updating the password.